### ISSUE TITLE
Sorting in panel creates duplicate sorting numbers

### ISSUE THREAD

**Issue by texnixe** [getkirby/panel#620]
Moved from #144 in getkirby.com repo

This usually happens with 20+ subpages, in this case there are around 70+ subpages. 
When trying to arrange one particular page in the edit-pages panel suddenly the order changed to

- 40
- 43
- 41
- 42
- 44
- 45

Looking into the content folder, the following directories can be found:

- 40-title…
- 40-title2…
- 41-title3…
- 42-title4…
- 42-title5…
- 43-title6…
- 44-title7…
- 45-title8…

The panel sorting order wasn't changed. 

See also the [forum discussion](https://forum.getkirby.com/t/sorting-in-the-panel-creates-duplicate-sorting-numbers/2401/4)

##Blueprint Main Page

````
<?php if(!defined('KIRBY')) exit ?>

title: Presse
pages:
  template: pressemeldung
files: false
fields:
  pageSettings:
    label: Seiteninhalt
    type: headline
    help: Normaler Seiteninhalt
  title:
    label: Title
    type:  text
  text:
    label: Text
    type:  textarea
  metaSettings:
    label: Meta
    type: headline
    help: Für Suchmaschinen & Co
  metaDescription:
    label: Beschreibung
    type: textarea
    buttons: false
    help: Kurze Beschreibung des Seiteninhaltes - 200 Character max
  metaKeywords:
    label: Schlagworte
    type:  textarea
    buttons: false
    help: Schlagworte zum Seiteninhalt - 200 Character max
```` 

### PHP-Code (Website Output)
````
<div class="cf">
  <?php foreach($data as $project): ?>
  <div class="subline">
    <h2><?php echo $project->title()->html() ?></h2>
    <?php echo $project->text()->kirbytext() ?>
  </div>
  <?php endforeach ?>
</div>
````

##  Blueprint Single Page

````
<?php if(!defined('KIRBY')) exit ?>

title: Pressemeldung
pages: false
files:
  sortable: true
fields:
  pageSettings:
    label: Seiteninhalt
    type: headline
    help: Normaler Seiteninhalt
  title:
    label: Title
    type:  text
  archiv:
    label: Archiv?
    type: Checkbox
    width: 1/2
    text: Soll der Artikel archiviert werden?
  text:
    label: Text
    type:  textarea
  metaSettings:
    label: Meta
    type: headline
    help: Für Suchmaschinen & Co
  metaDescription:
    label: Beschreibung
    type: textarea
    buttons: false
    help: Kurze Beschreibung des Seiteninhaltes - 200 Character max
  metaKeywords:
    label: Schlagworte
    type:  textarea
    buttons: false
    help: Schlagworte zum Seiteninhalt - 200 Character max
````

### PHP-Code (Website Output)
````
<div class="cf">
  <?php foreach($data as $project): ?>
    <?php if( $site->language() == $project->content()->language() ): ?>
      <?php echo "LANG: " . $site->language() . '/' . $project->content()->language(); ?>
  <div class="subline">
    <h2><?php echo $project->title()->html() ?></h2>
    <?php echo $project->text()->kirbytext() ?>
  </div>
    <?php endif ?>
  <?php endforeach ?>
</div>
````

**Comment by distantnative** [comment]
Can't reproduce this at all :/

**Comment by bastianallgeier** [comment]
Is that a 2.1 bug or 2.2?

**Comment by distantnative** [comment]
In the forum post it says 2.1.1 – so maybe fixed with 2.2 anyways?

**Comment by distantnative** [comment]
As this might be already fixed in the upcoming 2.2, I will close this issue for now. If it appears again in the 2.2, please open a new issue and let us know.
